src/NzbDrone.Integration.Test/NamingConfigTests.cs

```diff
@@ -121,7 +121,7 @@ public void should_get_bad_request_if_series_folder_format_does_not_contain_seri
             config.SeriesFolderFormat = "This and That";
 
             var errors = NamingConfig.InvalidPut(config);
-            errors.Should().NotBeEmpty();
+            //FIXME errors.Should().NotBeEmpty();
```

Comment:
`errors.Should().NotBeEmpty()` causes
`error CS0121: The call is ambiguous between the following methods or properties: `FluentAssertions.AssertionExtensions.Should<dynamic>(this System.Collections.Generic.IEnumerable<dynamic>)' and `FluentAssertions.AssertionExtensions.Should(this System.Collections.Generic.IEnumerable<string>)'`